app/commands/concept_exercise/create.rb

```diff
@@ -0,0 +1,19 @@
+class ConceptExercise
+  class Create
+    include Mandate
+
+    initialize_with :uuid, :slug, :title, :taught_concepts, :prerequisites, :deprecated, :git_sha, :synced_to_git_sha, :track
+
+    def call
+      ::ConceptExercise.create_or_find_by!(uuid: uuid, track: track) do |e|
+        e.slug = slug
+        e.title = title
+        e.taught_concepts = taught_concepts
+        e.prerequisites = prerequisites
+        e.deprecated = deprecated
+        e.git_sha = git_sha
+        e.synced_to_git_sha = synced_to_git_sha
```

Comment:
```suggestion
        e.attributes = attributes
        e.git_sha = e.synced_to_git_sha
```

Reply:
You might also want to consider moving that last line to into the `before_create` of the `Track` itself. We can discuss pros/cons of that together tomorrow :)